android/app/src/main/java/labs/pooh/mycanteen/ui/slideshow/SlideshowFragment.kt

```diff
@@ -1,28 +1,97 @@
 package labs.pooh.mycanteen.ui.slideshow
 
+import android.graphics.Bitmap
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.appcompat.content.res.AppCompatResources.getDrawable
+import androidx.core.view.isVisible
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import androidx.lifecycle.Observer
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+import io.reactivex.rxkotlin.addTo
+import io.reactivex.rxkotlin.toObservable
+import io.reactivex.schedulers.Schedulers
+import kotlinx.android.synthetic.main.app_bar_main.*
 import kotlinx.android.synthetic.main.fragment_slideshow.*
 import labs.pooh.mycanteen.R
+import labs.pooh.mycanteen.ui.view.RatedFoodView
+import labs.pooh.mycanteen.util.*
+import java.util.*
 
 class SlideshowFragment : Fragment() {
 
     private val slideshowViewModel: SlideshowViewModel by viewModels()
 
+    /**
+     * Resources that should be disposed when the fragment is destroyed
+     */
+    private val compositeDisposable = CompositeDisposable()
+
     override fun onCreateView(
             inflater: LayoutInflater,
             container: ViewGroup?,
             savedInstanceState: Bundle?
     ): View? {
         val root = inflater.inflate(R.layout.fragment_slideshow, container, false)
-        slideshowViewModel.text.observe(viewLifecycleOwner, Observer {
-            textSlideshow.text = it
-        })
         return root
     }
```

Comment:
Jest sens zapisywać zwracaną wartość do zmiennej?

Reply:
To wygenerowany kod,  na razie nie zmieniam bo i tak bedzie zmieniony